src/Onboarding/Setup/templates/index.html.php

```diff
@@ -27,34 +27,37 @@
 
     <!-- Configuration -->
     <?php
```

Comment:
Can we have a `@unreleased` tag to doc these changes?

Reply:
Resolved: c910945c0179f38fc191c6590a6917b98c679142